packages/releaser/.release-it.js

```diff
@@ -1,16 +1,28 @@
 const { addBangNotes } = require("conventional-changelog-conventionalcommits/utils");
-const { DEFAULT_COMMIT_TYPES } = require("conventional-changelog-conventionalcommits");
 
 const version = "${version}";
 const packageName = process.env.npm_package_name;
 const scope = packageName.split("/")[1];
+const COMMIT_TYPES = [
+  { type: "feat", section: "Features" },
+  { type: "fix", section: "Bug Fixes" },
+  { type: "refactor", section: "Code Refactoring" },
+  { type: "perf", section: "Performance Improvements" },
+  { type: "test",  hidden: true },
+  { type: "chore", hidden: true },
+  { type: "docs",  hidden: true },
+  { type: "style", hidden: true },
+];
```

Comment:
Could you pls add "build" and "ci" types. I've used them in the past in this repo and believe those are useful.

Reply:
we could use `chore(build)` and `chore(ci)` or you think it's better to have specialized category?

Reply:
Just checked out the comment with the spec. I don't think it's better or worse. It was there before so i used it. But chore would work too indeed.